### DISCUSSION TITLE
Create a server, Creates pages, Stop Server, Get errors on restart

### DISCUSSION THREAD

**Discussion by tmunsch** [requarks/wiki#5079] [Error / Bug Report]
Long story short I am a developer and I have noticed a huge issue. I created a Pterodactyl Egg of wiki.js and have been running it in my panel with no issues. Except when I restart the server. 

Keep in mind that pterodacytl is just using docker. 

After I restart I receive a sendfile absolute path error.

I followed the instructions on a thread to modify the assets.js file and It works but this brings up a few questions.

I noticed when I got the path wrong in the assets.js file the console stated that /opt/wiki/backups didn't exist

So is it pulling the files from my backups folder now? 
After i corrected the path to where my backups folder actually is everything came back up.

After making that change it seems to work fine but it's definitely a little slower. Some insight would be greatly appreciated.

P.S. I love this project!

OS: Debian Docker Image/Pterodactyl Panel
Wiki Version: 2.5.276
NodeJS: 16.14.0
SQLite: 3.34.0

Link to my Wiki: [Here](https://wiki.bofanodes.io/)

**Comment by NGPixel** [comment]
I don't quite understand your problem and you definitely shouldn't modify a js file. Can you elaborate?
